tests/setup.js

```diff
@@ -9,6 +10,25 @@ import '@testing-library/jest-dom/extend-expect';
 
 import 'amo/polyfill';
 
+if (process.env.TEST_DEBUG !== 'FULL') {
+  configure({
+    getElementError: (message) => {
+      const error = new Error(message.split('\n')[0]);
```

Comment:
I'm on the fence about this. 

On the one hand, it messes up the error stack, making errors harder to debug if you are not in full mode. On the other hand, most of the time we should be in full mode anyway?

Maybe we should not be messing with that and just set `DEBUG_PRINT_LIMIT=0` ? (Which unfortunately doesn't prevent testing library from displaying the list of roles in the document...)

We could also disable the stack completely (`error.stack = null` seems to do the trick) but I don't think that's much better...

Reply:
I tried running with `error.stack = null`, but I felt that that hid too much important information, and would make debugging any CI failures quite difficult, so I'd rather keep the error stack always. I also tried `DEBUG_PRINT_LIMIT=0`, but, as you say, it doesn't prevent the really long messages from testing library.

We would come up with a compromise and maybe output more than just the first line of the messages, e.g., print the first 5 lines. In most cases that would probably give us enough info to hint at the cause of a failure.

WDYT?